dndtool/lib/dndtool/player_cache.ex

```diff
@@ -0,0 +1,84 @@
+defmodule Player do
+
+  defstruct name:         "Steve the Potion Chugger",
+            strength:      0, dexterity:       0,
+            constitution:  0, wisdom:          0,
+            intelligence:  0, charisma:        0,
+            acrobatics:    0, animal_handling: 0,
+            arcana:        0, athletics:       0,
+            deception:     0, history:         0,
+            insight:       0, intimidation:    0,
+            investigation: 0, medicine:        0,
+            nature:        0, perception:      0,
+            performance:   0, persuasion:      0,
+            religion:      0, sleight_of_hand: 0,
+            stealth:       0, survival:        0
+
+end
+
+defmodule Player_Cache do
+
+  def start_link() do
+    Agent.start_link(fn -> load_players() end, name: __MODULE__ )
+  end
+
```

Comment:
This module seems a little complex for what it does. 

When you save a player, to convert the map into a list of tuples, then write them out with a ":" between the key and value.

Can't you do the same when you read it back?